client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -370,7 +388,10 @@ export const useCommandsArrayWpcom = ( {
 		{
 			name: 'manageStagingSites',
 			label: __( 'Manage staging sites' ),
-			callback: setStateCallback( 'manageStagingSites' ),
+			callback: setStateCallback(
+				'manageStagingSites',
+				__( 'Select site staging sites to manage' )
```

Comment:
This almost sounds like Yoda would speak so there is kind of a cool aspect to it but I am finding some of these sentence a bit confusing. It makes me pause and think twice e.g. does this sentence make sense?

I think something more straightforward might be a better fit e.g. `Select site to manage staging sites` - open to feedback from others though!